src/excmds.ts

```diff
@@ -2425,6 +2427,7 @@ export async function echo(...str: string[]) {
  */
```

Comment:
Please document `arg` :)

Reply:
You should also document the `//` trick for allowing it to receive an argument from `composite`.

@cmcaine: I'd like to hear what you think about `//`.

Reply:
I've implemented -p now, which means we don't need the `//` hack anymore.